pyproject.toml

```diff
@@ -127,7 +127,7 @@ contraints = [
     "PyYAML==6.0",
     "requests==2.28.2",
     "responses==0.23.1",
-    "rohmu==1.0.10",
+    "rohmu==2.3.0",
```

Comment:
Why are we bumping this version as well?

Reply:
i had to update this and other versions as they failed to download during ci